tests/tests_geomstats/test_wald_space.py

```diff
@@ -0,0 +1,246 @@
+import geomstats.tests
+from geomstats.stratified_geometry.wald_space import Split, Topology, Wald, WaldSpace
+from tests.conftest import Parametrizer
+from tests.data.wald_space_data import WaldSpaceTestData, WaldTestData
+from tests.stratified_geometry_test_cases import PointSetTestCase, PointTestCase
+
+
+class TestWaldSpace(PointSetTestCase, metaclass=Parametrizer):
+
+    _PointSet = WaldSpace
+    testing_data = WaldSpaceTestData()
+
+
+class TestWald(PointTestCase, metaclass=Parametrizer):
+
+    _Point = Wald
+    testing_data = WaldTestData()
+
+
+@geomstats.tests.np_only
+class TestSplit(geomstats.tests.TestCase):
```

Comment:
@JonasLueg, what are your thoughts on creating `SplitData` and make a better split (;)) between test and data as in the other cases? Do you think it would be too much complexity for such a simple case?

Reply:
Yes, I can certainly do that, the main motivation is probably the consistency with the testing framework.